.github/workflows/docker_build.yml

```diff
@@ -54,4 +54,18 @@ jobs:
           labels: ${{ steps.meta.outputs.labels }}
           build-args: |
             TAG=${{ env.RELEASE_VERSION }}
-            
\ No newline at end of file
+            
+      - name: Lowercase image name for trivy
+        id: string
+        uses: ASzc/change-string-case-action@v6
+        with:
+          string: ${{ env.IMAGE_PATH }}
+
+      - name: Run Trivy vulnerability scanner
+        uses: aquasecurity/trivy-action@master
+        with:
+          image-ref: '${{ steps.string.outputs.lowercase }}:${{ steps.meta.outputs.version }}'
+          format: 'table'
```

Comment:
can we have the report in a different format? Even the json format described here: https://aquasecurity.github.io/trivy/v0.17.2/examples/report/ is easier to read than the table format